app/views/content/terms_candidate.md

```diff
@@ -34,9 +34,13 @@ You can apply again to one course at a time as many times as you like.
 
 Once you submit your application it’s sent to your teacher training providers straight away.
 
-Contact us within 5 working days at <%= bat_contact_mail_to %> if you need to change something.
+Contact us within 5 working days at <%= bat_contact_mail_to %> if you need to change something. We can only update your:
 
-You can ask us to add courses if you have not used all your course choices, or change your personal details.
+* course choices
+* personal statement
+* qualifications 
```

Comment:
This is most of the application!? Does this mean that work experience cannot be update, or is that just less common? If anything can be updated (within 5 days) then maybe it’s simpler to say that, instead of listing sections?

Reply:
my understanding is that these are the only things that can be updated

Reply:
have asked Henna Malik to confirm

Reply:
turns out you can also update your work history... getting a bit confused by this support policy now!